build/common.sh

```diff
@@ -171,6 +171,10 @@ function kube::build::verify_prereqs() {
 function kube::build::docker_available_on_osx() {
   if [[ -z "${DOCKER_HOST}" ]]; then
     kube::log::status "No docker host is set. Checking options for setting one..."
+    if [[ -a "/Applications/Docker.app/Contents/MacOS/Docker" ]]; then
```

Comment:
I've only seen `-e` for "file exists", and `man test` on my linux machine only references `-a` for AND, but this seems to work, so LGTM.